apps/frontend/src/libs/components/balance-wheel-chart/styles.module.css

```diff
@@ -1,3 +1,7 @@
-.container {
+.container-animation {
```

Comment:
This maybe too picky of me, but I would write it like `.animation-container`, that was it sounds more like a container for animation